7.3.частые_сценарии/7.3.14.py

```diff
@@ -0,0 +1,6 @@
+flag = "YES"         
+for i in range(10):  
```

Comment:
Если переменная цикла не используется в теле цикла, тогда называй ее `_`:
```suggestion
for _ in range(10):
```